wp/lib/bap_wp/src/precondition.mli

```diff
@@ -34,6 +34,11 @@ module Env = Environment
 
 module Constr = Constraint
 
+(** A record of generated constraint lists mapped to assumptions and verification
```

Comment:
If this type is opaque, arguably we need to be *less* descriptive about the insides, and talk more about what a user needs to do with it...